### DISCUSSION TITLE
My friends, I have encountered several problems while using it and I am seeking help

### DISCUSSION THREAD

**Discussion by wolono** [fuma-nama/fumadocs#883] [Q&A] [answered]
## 1. The site name is missing from the head on the homepage
![image](https://github.com/user-attachments/assets/3afdb8ed-40d2-47d7-884c-66dd3f0b78f7)
## 2. Unable to load folder properly encountered in setting up Folder Group
![image](https://github.com/user-attachments/assets/437564c9-136c-4e43-bdf0-6b96b11e0ccd)
I deleted the default docs folder in the settings and created two new folders, but encountered a 404 error when accessing the newly created folders. According to the document, we have created metajson files in the respective folders. Below are the contents of the metajson files
### erpnext/meta.json
```
{
    "title": "ERPNext",
    "root": true,
    "pages": [
      "---Get Started---",
      "index.mdx",
      "test.mdx"
    ]
}
```
### framework/meta.json
```
{
    "title": "Framework",
    "root": true,
    "pages": [
      "---Get Started---",
      "index.mdx",
      "test.mdx"
    ]
  }
```
### Error message when accessing the page
![image](https://github.com/user-attachments/assets/cc149f3d-5d9e-4af8-afc0-3bed2f4a67b0)
Did I miss any other settings?
## Regarding Localization Settings
![image](https://github.com/user-attachments/assets/a8bfbb15-b8b1-48e6-83a6-85ba0154d09d)
I don't need multilingual support during use, so I don't want to set i18. Is there any other way to localize the content in these locations to the language I need?

## recommendation
Regarding the official document content, it is recommended to specify which files need to be modified or added in some content and provide the directory where they need to be located. This is more conducive for newcomers to read
### For example:
![image](https://github.com/user-attachments/assets/61611f79-cb74-4f4d-b010-d6599f78c8ba)
Do we need to modify an existing file or create a new file in that directory

I hope to receive everyone's help with the above questions. Thank you all

The suggestion is just my personal opinion, perhaps unnecessary. Please forgive any offense. thank you

**Comment by wolono** [comment]
My code repository
https://github.com/wolono/docs
website
https://docs.erpnext.work
The above problem code can be found in the repository

**Comment by fuma-nama** [comment, marked as answer]
1. Use Next.js Metadata API
2. You need to put the files inside `content/docs`, it is the default content location for docs.
3. As you said "localize", Fumadocs UI's i18n API can change the labels without the need to support multilingual, you can use `I18nProvider` to provide translations.
